code/modules/client/client_procs.dm

```diff
@@ -168,11 +168,11 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
 	if(CONFIG_GET(flag/automute_on) && !holder && last_message == message)
 		src.last_message_count++
 		if(src.last_message_count >= SPAM_TRIGGER_AUTOMUTE)
-			to_chat(src, span_danger("You have exceeded the spam filter limit for identical messages. An auto-mute was applied."))
+			to_chat(src, span_danger("You have exceeded the spam filter limit for identical messages. A mute was automatically applied for the current round. Contact admins to request its removal."))
 			cmd_admin_mute(src, mute_type, 1)
 			return TRUE
 		if(src.last_message_count >= SPAM_TRIGGER_WARNING)
-			to_chat(src, span_danger("You are nearing the spam filter limit for identical messages."))
+			to_chat(src, span_danger("You are nearing the auto-ban limit for identical messages."))
```

Comment:
```suggestion
			to_chat(src, span_danger("You are nearing the round auto-mute for identical messages."))
```

Reply:
![image](https://user-images.githubusercontent.com/7069733/154825685-e5e7a451-aa50-4741-982f-af132f655ef0.png)

Reply:
you should put that into a code comment honestly